client/src/playground/index.tsx

```diff
@@ -1,10 +1,12 @@
 import { useCallback, useEffect, useRef, useState } from "react";
 import { useSearchParams } from "react-router-dom";
 import useSWRImmutable from "swr/immutable";
-import prettier from "prettier/esm/standalone.mjs";
-import parserBabel from "prettier/esm/parser-babel.mjs";
-import parserCSS from "prettier/esm/parser-postcss.mjs";
-import parserHTML from "prettier/esm/parser-html.mjs";
+import prettier from "prettier/standalone.js";
+import prettierPluginBabel from "prettier/plugins/babel.js";
+import prettierPluginCSS from "prettier/plugins/postcss.js";
+// XXX Using .mjs until https://github.com/prettier/prettier/pull/15018 is deployed
+import prettierPluginESTree from "prettier/plugins/estree.mjs";
+import prettierPluginHTML from "prettier/plugins/html.js";
```

Comment:
Sorry, I meant drop the extension altogether, like:

```suggestion
import prettier from "prettier/standalone";
import prettierPluginBabel from "prettier/plugins/babel";
import prettierPluginCSS from "prettier/plugins/postcss";
// XXX Using .mjs until https://github.com/prettier/prettier/pull/15018 is deployed
import prettierPluginESTree from "prettier/plugins/estree.mjs";
import prettierPluginHTML from "prettier/plugins/html";
```